analysis/neutron/openmc_model.py

```diff
@@ -155,8 +157,22 @@ def baby_geometry(x_c: float, y_c: float, z_c: float):
     ######## Sphere #################
     sphere = openmc.Sphere(x0=x_c, y0=y_c, z0=z_c, r=50.00)  # before r=50.00
 
+    ######## RectangularParallelepiped #################
+    rectangular_1 = openmc.model.RectangularParallelepiped(
+        -17.5 + x_c, -9.5 + x_c, -8 + y_c, +8 + y_c, z_c - 28.00, z_c - 24.00
+    )
+    rectangular_2 = openmc.model.RectangularParallelepiped(
+        -8.5 + x_c, -0.5 + x_c, -8 + y_c, +8 + y_c, z_c - 28.00, z_c - 24.00
+    )
+    rectangular_3 = openmc.model.RectangularParallelepiped(
+        32.5 + x_c, 40.5 + x_c, -8 + y_c, +8 + y_c, z_c - 28.00, z_c - 24.00
+    )
+    rectangular_4 = openmc.model.RectangularParallelepiped(
+        23.5 + x_c, 31.5 + x_c, -8 + y_c, +8 + y_c, z_c - 28.00, z_c - 24.00
+    )
+
```

Comment:
I see a lot of magic numbers here. For example, I feel like `28.00` means something: is it the bottom of the solid?
Is the height of the solid 4 cm? in that case, you could have a variable `height = 4.0` and then use it in the construction. Same for width and thickness

This would increase readability